apps/abstraxion-dashboard/components/Overview.tsx

```diff
@@ -1,11 +1,13 @@
 import { formatBalance, getCommaSeperatedNumber } from "@/utils";
 import { useAccountBalance } from "@/hooks/useAccountBalance";
-import { RightArrowIcon } from "./Icons";
+import { RightArrowIcon, ScanIcon } from "./Icons";
 import { WalletSend } from "./WalletSend/WalletSend";
+import { WalletReceive } from "./WalletReceive";
+import { AbstraxionAccount } from "@/hooks";
 
 export const XION_TO_USDC_CONVERSION = 50;
 
-export const Overview = () => {
+export const Overview = ({ account }: { account?: AbstraxionAccount }) => {
```

Comment:
Is there a reason for pulling this in as prop as opposed to getting value from context or useAccount hook?

Reply:
Not necessarily a huge reason for one over the other. I could switch it so there is less prop drilling, was just following what was already done in AccountInfo component.